i18n/README.de-de.md

```diff
@@ -0,0 +1,326 @@
+spaltenorientiertes Speichermodell, parallelisierte Vektorausführung, SIMD-Anweisungen,
+und Low-Latency-Techniken. Die gesamte Codebasis ist von Grund auf neu aufgebaut
+in Java und C++, ohne Abhängigkeiten und ohne Garbage Collection.
+
+<div align="center">
+  <a href="https://demo.questdb.io">
+    <img alt="Die QuestDB-Webkonsole zeigt mehrere SQL-Anweisungen und visualisiert eine Abfrage als Diagramm" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  </a>
+</div>
+
+## Probieren Sie QuestDB aus
+
+Wir stellen eine [Live-Demo](https://demo.questdb.io/) bereit, die mit den neuesten QuestDB Release
+und Beispieldatensätze bereitgestellt wird:
+
+- 10 Jahre Taxifahrten in NYC mit 1,6 Milliarden Reihen
+- Live-Handelsdaten von einer Kryptowährungsbörse
+- Geolokalisierung von 250.000 einzigartigen Schiffen im Laufe der Zeit
+
+## Loslegen
+
+### QuestDB installieren
+
+Um QuestDB auszuführen, kann Docker verwendet werden, um schnell loszulegen:
+
+```bash
+docker run -p 9000:9000 -p 9009:9009 -p 8812:8812 questdb/questdb
+```
+
+macOS-Benutzer können Homebrew verwenden:
+
+```bash
+brew install questdb
+brew services start questdb
+
+questdb start // To start questdb
+questdb stop  // To stop questdb
+```
+
+Die [QuestDB-Downloadseite](https://questdb.io/get-questdb/) bietet direkte
+Downloads für Binärdateien und enthält Details für andere Installationen und
+Methoden.
+
+### Verbinden Sie sich mit QuestDB
+
+Sie können mit QuestDB über die folgenden Schnittstellen interagieren:
+
+- [Web Console](https://questdb.io/docs/develop/web-console/) für interaktiv
+  SQL-Editor auf Port `9000`
+- [InfluxDB line protocol](https://questdb.io/docs/reference/api/influxdb/) zur
+  Aufnahme mit hohem Durchsatz am Port `9009`
+- [REST API](https://questdb.io/docs/reference/api/rest/) am port `9000`
+- [PostgreSQL wire protocol](https://questdb.io/docs/reference/api/postgres/) am
+  port `8812`
+
+### Daten einfügen
+
+Nachfolgend finden Sie unsere offiziellen InfluxDB-Leitungsprotokollclients für beliebte Programmier-
+Sprachen:
+
+- [.NET](https://github.com/questdb/net-questdb-client)
+- [C/C++](https://github.com/questdb/c-questdb-client)
+- [Go](https://pkg.go.dev/github.com/questdb/go-questdb-client)
+- [Java](https://questdb.io/docs/reference/clients/java_ilp/)
+- [NodeJS](https://questdb.github.io/nodejs-questdb-client)
+- [Python](https://py-questdb-client.readthedocs.io/en/latest/)
+- [Rust](https://docs.rs/crate/questdb-rs/latest)
+
+## Vergleich von QuestDB mit anderen Open-Source-TSDBs
+
+[Dieser Artikel](https://questdb.io/blog/2021/07/05/comparing-questdb-timescaledb-influxdb/)
+vergleicht QuestDB übergreifend mit anderen Open-Source-Timeseriesdatenbanken
+Funktionalität, Reife und Leistung.
+
+Hier sind hohe Kardinalität
+[Zeitreihen-Benchmark-Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
```

Comment:
```suggestion
[Timeseries-Benchmark-Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
```

Reply:
Requested one more change for row 108. Besides that, the German translation looks good.